stories/components/LessonCard.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import LessonCard from '../../components/LessonCard'
+
+export default {
+  component: LessonCard,
+  title: 'Components/LessonCard'
+}
+
+const numberOfChallenges = 10
+const numberOfLessons = 10
+const numberOfHours = 10
+
+export const Basic: React.FC = () => (
+  <LessonCard
+    coverImg="Name of Image File"
+    title="Title of Lesson"
+    challengeCount={numberOfChallenges}
+    lessonCount={numberOfLessons}
+    hourCount={numberOfHours}
+    description="Lesson Description"
+  ></LessonCard>
```

Comment:
minor, could you hardcode one of the lessons so the story looks better? https://deploy-preview-55--youthful-thompson-448332.netlify.com/?path=/story/components-lessoncard--basic

Reply:
Fixed 👍